components/feature/autofill/src/main/java/mozilla/components/feature/autofill/ui/AbstractAutofillUnlockActivity.kt

```diff
@@ -40,7 +41,10 @@ abstract class AbstractAutofillUnlockActivity : FragmentActivity() {
         val structure: AssistStructure? = intent.getParcelableExtra(AutofillManager.EXTRA_ASSIST_STRUCTURE)
 
         // While the user is asked to authenticate, we already try to build the fill response asynchronously.
-        fillResponse = lifecycleScope.async(Dispatchers.IO) { fillHandler.handle(structure, forceUnlock = true) }
+        fillResponse = lifecycleScope.async(Dispatchers.IO) {
+            val builder = fillHandler.handle(structure?.toRawStructure(), forceUnlock = true)
```

Comment:
Same comment about `null` structure.

Also, do you care about asserting that we actually get a structure? Not worth crashing over, but could be useful to perhaps log this as a `w`, at least, to maybe help w/ debugging weird issues.